sources/packages/backend/libs/services/src/sfas/sfas-application.service.ts

```diff
@@ -74,7 +77,9 @@ export class SFASApplicationService extends DataModelService<SFASApplication> {
       .createQueryBuilder("sfasApplication")
       .select("SUM(sfasApplication.bslAward)")
       .innerJoin("sfasApplication.individual", "sfasFTstudent")
-      .where("sfasFTstudent.id = :studentId", { studentId })
+      .innerJoin("sfasFTstudent.student", "simsStudent")
```

Comment:
As a recommendation, the below would be more aligned with what we use (the table name as an alias).
```ts
const total = await this.repo
      .createQueryBuilder("sfasApplication")
      .select("SUM(sfasApplication.bslAward)")
      .innerJoin("sfasApplication.individual", "individual")
      .innerJoin("individual.student", "student")
      .where("sfasApplication.applicationCancelDate IS NULL")
      .andWhere("student.id = :studentId", { studentId })
      .getRawOne<{ sum?: number }>();
```
Alternatively, sometimes it works without the inner join also.
```ts
const total = await this.repo
      .createQueryBuilder("sfasApplication")
      .select("SUM(sfasApplication.bslAward)")
      .innerJoin("sfasApplication.individual", "individual")
      .where("sfasApplication.applicationCancelDate IS NULL")
      .andWhere("individual.student.id = :studentId", { studentId })
      .getRawOne<{ sum?: number }>();`
```

Reply:
I am picking option 1.